<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneBook</class>
 <widget class="QMainWindow" name="PhoneBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhoneBook</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>907</width>
      <height>553</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>907</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1_signin">
     <widget class="QLineEdit" name="login_userId">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>214</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>userId</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>121</width>
        <height>109</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>131</width>
        <height>109</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User PW</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_userPw">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>214</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>userPw</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signin">
      <property name="geometry">
       <rect>
        <x>274</x>
        <y>300</y>
        <width>340</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="forgetAccount">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>360</y>
        <width>160</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Forget Account</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signup">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>360</y>
        <width>160</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Sigin up</string>
      </property>
     </widget>
     <widget class="QLabel" name="signin_userPwLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>260</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="signin_userIdLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>160</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2_findAccount">
     <widget class="QLineEdit" name="findPw_pwInput">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>511</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Enter Id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submit">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>220</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>비밀번호 찾기</string>
      </property>
     </widget>
     <widget class="QLabel" name="find_account_pw_lb">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>501</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="find_account_back_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
     <widget class="QLabel" name="findPw_pwLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page3_signup">
     <widget class="QLabel" name="signup_user_id_lb">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>아이디</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signup_user_id_input">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>user_id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signup_user_id_dp_chk_btn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>100</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>아이디 중복 확인</string>
      </property>
     </widget>
     <widget class="QLabel" name="signup_user_id_lb_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>비밀번호</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signup_user_pw_input">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>user_pw</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signup_user_pw_double_input">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>user_pw</string>
      </property>
     </widget>
     <widget class="QLabel" name="signup_user_id_dp_chk_lb">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="signup_submit_btn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>460</y>
        <width>471</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>가입완료</string>
      </property>
     </widget>
     <widget class="QLabel" name="signup_user_id_lb_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>전화번호</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signup_user_phone_number_input">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>user_phone</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signup_back_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
     <widget class="QLabel" name="signup_pw_chk_lb">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>330</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="signup_phone_number_chk_lbl">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>430</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
